package.json

```diff
@@ -19,15 +19,12 @@
     "examples": "webpack-dev-server --config examples/webpack.config.js --no-info --content-base examples/"
   },
   "license": "MIT",
-  "peerDependencies": {
-    "react": "0.12.x"
-  },
   "dependencies": {
+    "react": "0.12.x",
```

Comment:
Hmm... so it looks like Radium doesn't have a dist build yet. So here's my understanding (which may be very wrong) -- for things that you use internally and want added, add a `dependencies` entry, for things where you are the add-on, add a `peerDependencies` and `devDependencies` for your dev stuff. The reasoning is that your users are choosing React first, and then something like Radium as the add-on.

... but I'm not sure about how the packaging actually matters. Maybe one of us (or someone else at Formidable who's interested) see what the canonical / most typical answer is?

For a contrast, in `chai-jq`, I do a UMD wrapper: https://github.com/FormidableLabs/chai-jq/blob/master/chai-jq.js#L500-L528 but actually do not **not** have either `peerDependencies` or `dependencies` on Chai or jQuery, even though both are required for the package... https://github.com/FormidableLabs/chai-jq/blob/master/package.json